### DISCUSSION TITLE
Home Assistant e media server

### DISCUSSION THREAD

**Discussion by Phoe95** [indomus/forum#3254] [HUB: Home Assistant]
Ciao a tutti, vorrei iniziare a domotizzare casa tramite Home Assistant ed ho acquistato un mini pc con processore Intel N95, 16 GB di RAM LPDDR5 e 512 GB di SSD. Mi sembra di capire che di base la migliore soluzione sia installare Home Assistant Core su Docker ma, siccome le prestazioni del dispositivo mi sembrano sprecate per fa girare solo HA, volevo sfruttarlo anche come media server per evitare di tenere un ulteriore dispositivo acceso H24. Vorrei utilizzare il mini PC per poter condividere in contemporanea i miei file multimediali nella rete locale e vorrei capire se in questo caso il Core su Docker rimanga comunque la scelta migliore e soprattutto quali programmi potrei installare in parallelo ad Home Assistant per gestire i file multimediali. Attualmente ho collegato un hard disk esterno al Fritz!Box e l'ho condiviso in rete locale tramite la funzione Fritz!NAS ma il mini PC che ho preso mi sembra molto più adatto di un router a svolgere questa funzione e vorrei approfittarne per creare un sistema più funzionale come, ad esempio, un media server o un NAS.

**Comment by lux90ita** [comment]
io andrei su proxmox 
ma devi saperti muovere

**Comment by mimelaine** [comment]
Personalmente ho installato OpenMediaVault come NAS, poi con il compose (che ha dentro come plugin) ho tirato su il docker per HA core e PLEX come media server. In questo modo avresti un vero NAS, e tramite il compose di docker tutto quello che ti serve tirare su in pochi click.

Al netto delle competenze tecniche (OMV è davvero molto semplice da gestire e la documentazione/forum sono molto attivi e aggiornati) le prestazioni con Proxmox sarebbero inferiori ed è un controsenso se non hai delle necessità particolari di virtualizzazione.
